bigquery_schema_generator/generate_schema.py

```diff
@@ -694,8 +694,7 @@ def main():
         default=1000)
     parser.add_argument(
         '--debugging_map',
-        help=
-        'Print the metadata schema_map instead of the schema for debugging',
+        help='Print the metadata schema_map instead of the schema for debugging',
```

Comment:
This is longer than 80 columns, and will revert to the original form by the autoformatter when I run it. Let's make it slightly shorter:

```suggestion
        help='Print the metadata schema_map instead of the schema',
```